src/ipld/util.rs

```diff
@@ -117,10 +117,11 @@ where
     T: Future<Output = anyhow::Result<Vec<u8>>> + Send,
 {
     let estimated_total_records = estimated_total_records.unwrap_or_default();
-    let bar = ProgressBar::new(estimated_total_records);
-    bar.message(progress_bar_message.unwrap_or("Walking snapshot "));
-    bar.set_units(progress_bar::Units::Default);
-    bar.set_max_refresh_rate(Some(Duration::from_millis(500)));
+    let message = progress_bar_message.unwrap_or("Walking snapshot");
+    let bar = Arc::new(Mutex::new(ProgressLog::new(
+        message,
+        estimated_total_records,
+    )));
```

Comment:
To mimic the old progress bars, we could put the Arc+Mutex inside `ProgressLog`.

Reply:
Yes.